qiskit/utils/mitigation/_filters.py

```diff
@@ -112,6 +111,7 @@ def apply(self, raw_data, method="least_squares"):
         """
         from scipy.optimize import minimize
         from scipy import linalg as la
+        from .. import parallel_map  # pylint: disable=cyclic-import
```

Comment:
To be fair to pylint, from within the same package, it's probably not wrong to complain about this - it probably should be `from ..parallel import parallel_map` to avoid the true import cycle that `QuantumInstance` otherwise generates; `qiskit.utils` can't be fully initialised before `qiskit.utils.mitigation`.

Reply:
I adjusted this in: https://github.com/Qiskit/qiskit/pull/11514/commits/9e310197968d6a2d0fb4da26e696320b48013a3e